### ISSUE TITLE
Encoding issue for abbreviations of SpecificWeight

### ISSUE THREAD

**Issue by jnyrup** [angularsen/UnitsNet#330]
There seems to be an encoding issue for the generated abbreviations for SpecificWeight where an `Â` is inserted right before the `³`.

```
new CulturesForEnumValue((int) SpecificWeightUnit.KilogramForcePerCubicCentimeter,
    new[]
    {
        new AbbreviationsForCulture("en-US", "kgf/cmÂ³"),
    }),
```

**Comment by angularsen** [comment]
Good catch, I see a bunch of them are all wrong. It's likely an encoding issue with this particular JSON file.

**Comment by angularsen** [comment]
Fixed in #326
